DEBUG:root:Order:2 5 50.0
DEBUG:root:Finish turn

ERROR:root:Error in bot.
Traceback (most recent call last):
  File "/workspaces/CMPM146-P3/opponent_bots/easy_bot.py", line 37, in <module>
    current_line = input()
EOFError: EOF when reading a line
